tools/initMigrationToPP/initMigrationToPP.json.example

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "Timelock",
+    "polygonRollupManagerAddress": "0x0B306BF915C4d645ff596e518fAf3F9669b97016",
+    "rollupID": 4,
+    "newRollupTypeID": 2,
+    "timelockDelay": 0,
+    "deployerPvtKey": "",
```

Comment:
Better remove deployerPvtKey from this file for security and force to use `.env`